build/common/entrypoint.sh

```diff
@@ -56,4 +56,6 @@ fi
 
 export HOME=/home/"$username"
 
+gosu "$LOCAL_UID:$LOCAL_GID" "/git-setup.sh"
+gosu "$LOCAL_UID:$LOCAL_GID" "/ssh-setup.sh"
```

Comment:
build.sh has the ability to add extra hosts to the ssh setup. Is it ok to remove that support?

Reply:
Ah, yeah, I forgot to fix that bit. Change to come.

Reply:
Addressed in https://github.com/ARMmbed/mbl-tools/pull/359/commits/b6a74ba26aa42ff9d11c6b13d800f2a54ff46120 and later cleanup commits.